dex/overlays/moc/smaug/dex-cm.yaml

```diff
@@ -0,0 +1,68 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dex
+data:
+  config.yaml: |
+    issuer: http://dex-dex.apps.moc-infra.massopen.cloud
+
+    storage:
+      type: memory
+
+    web:
+      http: "0.0.0.0:5556"
+
+    grpc:
+      addr: "0.0.0.0:5557"
+
+    telemetry:
+      http: "0.0.0.0:5558"
+
+    oauth2:
+      skipApprovalScreen: true
+
+    staticClients:
+      - id: argo
+        name: Argo
+        redirectURIs:
+          - http://argo-server-opf-argo.apps.moc-infra.massopen.cloud/oauth2/callback
+        secretEnv: ARGO_SECRET
+
+      - id: grafana
```

Comment:
let's just keep this client(`grafana`) as an example and remove all other clients from this file, since we will need to update all the clients once they are deployed again in the smaug cluster.